### ISSUE TITLE
"ImageNet" Classification doesnt work

### ISSUE THREAD

**Issue by Zegervg** [ultralytics/ultralytics#6593]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

from ultralytics import YOLO

# Load a model
model = YOLO('yolov8n-cls.pt')  # load a pretrained model (recommended for training)

# Train the model
results = model.train(data='imagenet', epochs=100, imgsz=224)

will output bunch of errors

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@Zegervg hello! Thank you for reaching out and providing details about the issue you're experiencing with ImageNet classification using the YOLOv8 model.

Before we dive into solving the error you're facing, it's important to make sure that you're using the latest version of the YOLOv8 repo, as we frequently update it with fixes and improvements. Please ensure that you've pulled the latest updates.

When it comes to training with ImageNet, there are specific settings and requirements that need to be in place, such as the correct dataset format and configuration files. Given that YOLOv8 excels in tasks like detection and segmentation, training it directly on ImageNet (which is primarily used for classification tasks) might require extra steps or a detailed understanding of dataset preparation.

If after ensuring that you're on the latest version and you've correctly set up the ImageNet dataset, you still encounter errors, please make sure to check our documentation for any nuances or special instructions related to training on ImageNet with YOLOv8 models.

Without the specific error messages, it's challenging to provide a precise solution. However, I would suggest validating the dataset configuration and paths, making sure any necessary preprocess steps are followed (such as annotations or labels setup for the classification task), and ensuring your environment meets all the dependencies.

If you continue facing issues, please open a new issue and provide the full error logs and a more detailed explanation of the steps you've followed. This information will be crucial for our team to help you troubleshoot the problem effectively.

Remember, we're here to support you, and the YOLO community is always eager to assist with these challenges. 😊

**Comment by glenn-jocher** [comment]
@Zegervg works correctly for me. ImageNet is a massive dataset of a million images, make sure you have space and a strong internet connection, download should take maybe 12-24 hours.

<img width="1005" alt="Screenshot 2023-11-27 at 18 43 44" src="https://github.com/ultralytics/ultralytics/assets/26833433/e58c3cf2-b594-42b6-89c0-d30a4bac47b3">
